docs/cli/commands/logout.md

```diff
@@ -0,0 +1,29 @@
+---
+title: nest logout
+sidebar_label: logout
+---
+
+Remove an existing user account.
+
+```shell script
+nest login [username]
```

Comment:
```sh
nest logout [username]
     ^^^^^^
```

Reply:
Thanks for seeing this 👍 
copy-paste mistake